test/verify/check-blueprint

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python3
+
+# checkpoint:
+# 1. create blueprint
+# 2. blueprint filter
+# 3. blueprint sort
+
+import parent
+import testlib
+
+
+@testlib.nondestructive
+class TestBlueprint(testlib.MachineCase):
+    def setUp(self):
+
+        super().setUp()
+        self.allow_journal_messages(",".join(parent.allowed_journal_messages))
```

Comment:
this is wrong. It will pass just one string. You want to do `self.allow_journal_messages(*parent.allowed_journal_messages)`

Reply:
ack! Thanks.